src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/EventListener.java

```diff
@@ -251,6 +257,42 @@ protected Job asJob() {
     protected ParametersAction createParameters(GerritTriggeredEvent event, Job project) {
```

Comment:
NIT: you may want to update the javadoc of the method mentioning that now the `parametersAction` add the safe parameters to the default ones since it already has a description that says `Creates a ParameterAction and fills it with the project's default parameters + the Standard Gerrit parameters`.